rundk/README.md

```diff
@@ -35,12 +35,12 @@ Usage of rundk:
   --use-local-kubeconfig
       Use the same kubeconfig as local.
       It can be set either by setting env var KUBECONFIG or from ~/.kube/config
-  -mounts source1:target1,source2:target2,source3:target3
+  --mounts source1:target1,source2:target2,source3:target3
       A list of extra folders or files separated by comma that need to be mounted to run the test flow.
       It must be in the format of source1:target1,source2:target2,source3:target3.
-  -mandatory-env-vars string
-      A list of env vars separated by comma that must be set on local. (default "GOOGLE_APPLICATION_CREDENTIALS")
-  -optional-env-vars string
+  --mandatory-env-vars string
+      A list of env vars separated by comma that must be set on local.
```

Comment:
You are missing the important text that said environment variables will be promoted to the container

Reply:
Added.